### ISSUE TITLE
Keros Error  :  add Maximum number of element from Picker. After that Kero's give the Error

### ISSUE THREAD

**Issue by v-jashr** [microsoft/fluentui#12419]
- When i Add maximum element from Picker.

- Test with Kero's it gives the error

Can you please provides solution.

This Error  Occur on Office fabric official website too.
https://developer.microsoft.com/en-us/fabric#/controls/web/pickers

please refer following screen shot 

<img width="909" alt="Error 1" src="https://user-images.githubusercontent.com/50479173/77536251-1435c380-6ec2-11ea-92a1-f8ac898b0332.PNG">

![image](https://user-images.githubusercontent.com/50479173/77536235-0d0eb580-6ec2-11ea-8f11-a6ecd7071c41.png)

**Comment by aneeshack4** [comment]
Hi @v-jashr, thanks for reporting this. I'm able to repro this:
![image](https://user-images.githubusercontent.com/4161791/77560969-19781b80-6e8c-11ea-9a09-cd9e4f8a1f60.png)

If you inspect and look at the DOM, you can see it's technically a ComboBox which normally has an `<input>` but it's actually presented as a SelectionZone with a BasePicker inside which doesn't get formed explicitly as an `<input>` and instead is just a div (which has the `role='presentation'` with the BasePicker and TagItems inside each having `role='listitem'`). 

I think to fix the first error, we can add an `aria-label` to this [FocusZone inside BasePicker](https://github.com/microsoft/fluentui/blob/master/packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx#L285). What do you think @joschect? And what should that aria-label be?

I'm not sure about the second error - I don't think it's appropriate here because there's no `<input>` in the DOM structure. The ARIA spec says that would be the required textbox inside a combobox:
![image](https://user-images.githubusercontent.com/4161791/77562041-804a0480-6e8d-11ea-996e-9527d5f955a1.png)

DOM:
![image](https://user-images.githubusercontent.com/4161791/77560294-45df6800-6e8b-11ea-9277-5c1186e5dfd7.png)

**Comment by joschect** [comment]
@aneeshack4 I think the solution might be to not remove the input and disable it instead. Currently in the basepicker there is [this line](https://github.com/microsoft/fluentui/blob/ccc2e9a086d41a1674da17ad7f31d31fd84fe9a5/packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx#L305) which conditionally renders the input if it can add items. I believe that's what's causing this error because the combobox control is now missing it's required child (input) and there is no longer an input field to have a name.

**Comment by v-sagsah** [comment]
@aneeshack4 / @joschect can you please provide an update ?

**Comment by v-sagsah** [comment]
@aneeshack4  / @joschect please let us know, if this is a bug or is there any workaround to fix the issue ?
I would be grateful for your prompt reply.
